docs/loadcsv.adoc

```diff
@@ -250,8 +251,27 @@ The usage of `apoc.load.xls` is similar to apoc.load.csv with the main differenc
 
 You can either select the sheet by name like `'Kids'`, or offset like `'Results!B2:F3'`
 
-=== Examples for apoc.load.xls
+`CALL apoc.load.xls({url}, {Name of sheet}, {config})`
+
+The `{config}` parameter is a map
+
+[opts=header,cols="m,m"]
+|===
+| name | description
+| mapping | {mapping:{'<sheet>':{type:'<type>', dateFormat: '<format>', dateParse: [<formats>]}}}
+| <sheet> | name of the sheet
+| <type> | Default `String`, The type of the conversion requested (`STRING`, `INTEGER`, `FLOAT`, `BOOLEAN`, `NULL`, `LIST`, `DATE`, `DATE_TIME`, `LOCAL_DATE`, `LOCAL_DATE_TIME`, `LOCAL_TIME`, `TIME`)
+| dateFormat: <format> | Convert the Date into String (only String is allowed)
+| dateParse: [<formats>] | Convert the String into Date (Array of strings are allowed)
+|===
+
+==== Note
+
+In dateParse the first format matched return the date formatted, otherwise it will return an error
 
+In `format` config you can use the pattern describe as the Temporal functions: link:#_temporal[temporal funcionts]
+
+=== Examples for apoc.load.xls
```

Comment:
can you add a concrete example for date type(s) and one for dateParse/dateFormat?

Reply:
Sure